integrations/jira/.port/resources/port-app-config.yaml

```diff
@@ -1,6 +1,38 @@
 createMissingRelatedEntities: true
 deleteDependentEntities: true
 resources:
+  - kind: board
+    selector:
+      query: "true"
+    port:
+      entity:
+        mappings:
+          identifier: .id
+          title: .name
+          blueprint: '"jiraBoard"'
+          properties:
+            url: .self
+            type: .type
+          relations:
+            project: .location.projectId
```

Comment:
It does look like we don't need this relation since the boards API does not return the project (and you can have multiple projects in a board). Let's move this relation to the project blueprint instead

```json
{"maxResults":2,"startAt":1,"total":5,"isLast":false,"values":[{"id":84,"self":"https://your-domain.atlassian.net/rest/agile/1.0/board/84","name":"scrum board","type":"scrum"},{"id":92,"self":"https://your-domain.atlassian.net/rest/agile/1.0/board/92","name":"kanban board","type":"kanban"}]}
```